app/assets/javascripts/application.js

```diff
@@ -31,6 +31,7 @@
 //= require automate_import_export
 //= require dialog_field_refresh
 //= require excanvas
+//= require controllers/openshift_deployment/openshift_deployment_controller
```

Comment:
why is there a need to specify this controller explicitly? 
afaik all controllers together are loaded because of this line (in the same application.js file):
//= require_tree ./controllers/